tests/unit-core/test-mem-stats.c

```diff
@@ -16,46 +16,39 @@
 #include "jerryscript.h"
 #include "test-common.h"
 
-#ifdef JMEM_STATS
 
-const char *test_source = (
+int main (void)
+{
+  if (jerry_is_feature_enabled (JERRY_FEATURE_MEM_STATS))
```

Comment:
IMHO: It would be better to reverse the logic: check if the feature is not enabled and if so return with 0 (early exit). That way you do not need to indent the whole code in the main.